City-of-Helsinki/helsinki-paatokset
# Pp 217 platform update 2021 10 15

Updates:

Update helfi_platform_config to 2.0
Update Tunnistamo to 2.0
Fix issue with main navigation level 2 block

To test:
- Run `make new`
- Click around the site and see that everything still works
- Check that sides with main navgation level 2 block load correctly, eg. https://helsinki-paatokset.docker.so/en/resident-parking-permit-renewal
